modules/signing_worker/files/update-requirements.sh

```diff
@@ -15,17 +16,39 @@ popd &>/dev/null || exit
 # be different
 cwd=$(pwd)
 
+if [ -z "$1" ]; then
+    echo "Updating all requirement files"
+    for f in ${MY_DIR}/requirements.*; do
+        target=$(echo $f | awk -F '.' '{print $2}')
+        echo "Updating ${f}"
+        ${MY_DIR}/${0} $target $f
+    done
+    exit
+fi
+
+output_file="requirements.${1}.txt"
+
 trap 'cd $cwd' EXIT
 
 worker_type=$1
```

Comment:
Oh, that answers my question about `$1` 🙂 Then let's move this line before we test the content of `$1`